### DISCUSSION TITLE
死水真的阴吗

### DISCUSSION THREAD

**Discussion by sishui-buyin** [share121/inter-knot#1234] [其他]
![2ed72ntq](https://github.com/user-attachments/assets/2bc6ce02-8b3f-4340-9c57-27cd44525ea3)
 在今天我在玩火影忍者的时候看到有人说死水是阴间忍者，我就好奇了，波风水门（秽土转生）这么阳间的一个忍者却被他们说阴，我就好奇了，他们玩过这个忍者吗？
1.要伤害没伤害
2.要抓取普攻还没抓取
3.cd还长
4.还不容易攒能量
就这忍者也就能用插标瞬身机制玩了，他真的阴吗

**Comment by mixue4646** [comment]
不是死水还阴啊

**Comment by gaojiaobei** [comment]
我认为死水是一个极其需要尊重的老角色，因为他在火影忍者手游存在的这八年内，给这个游戏带来了巨大的热度，如果没有死水，火影忍者手游可能就没有今天，所以死水应该是火影手游全体尊重的对象，死水不但制作精良，而且特别需要操作的手法，如果段位没有3000分玩这个忍者就不能发挥他的实力，一但不能完全发挥实力，这个忍者就是无比鸡肋的忍者，此时需要的就是意识和手法来弥补对这个角色的操作不足，当技术大成时即可打出号称游戏最需要操作的玩法，人们都称为z字闪，这个忍者真的没有你们想的那么简单，不仅仅是在游戏中，在动漫中死水也是一个沉稳伟大的父亲，虽然没给鸣人一个完整的童年，不过他在战场上，在村子中，都是耀眼的领袖，就和游戏中需要操作的阳间战神一样，黄色闪光，你抓不住
死水就是火影的宝藏，火影的灵魂，决斗场里的VIP，副本中的铁人。无知时诋毁死水，懂事时喜欢死水，成熟时成为死水，玩死水说明这个人有大毅力大智慧。玩死水，说明这个人文化水平和素质修养都很高，玩死水，事业和爱情必双丰收，就像一条腾飞的巨龙。
其实我以后在火影忍者手游里，我以后再也不敢说秽土水门这个忍者的任何坏话，我承认死水这个忍者确实是要技术的，cd久的，阳间的，老忍者的，没抓取的，没反手的，被起手玩不了的，没有通灵秘卷t3的，伤害低的，要博弈的，z闪是难触发的，求求你们不要再攻击我了，我还不想在这个世界上消失
今天死水免费，我就玩了几把死水，感觉死水只能算T1，操作难度大，平a里没有带抓取的，技能是打不到霸体忍者的，要是不带上破霸体的密卷面对霸体忍者怕不是只能投降，谁能告诉我，死水为啥T0？因为加上散这个密卷就变强了？

**Comment by llxcx4** [comment]
不是哥们死水还阴啊

**Comment by gcj0418** [comment]
死水还阴啊

**Comment by xichulangren** [comment]
死水吗，有点意思哈

**Comment by Yukio4869** [comment]
怎么这里还有村里人，有点意思哈

**Comment by 23156484537** [comment]
死水不阴我吃千代大雷

**Comment by sjk428** [comment]
不是，死水还阴啊

**Comment by sishui-buyin** [comment]
看来各位都非常理智

**Comment by shenguojia123** [comment]
不认识，我见微知著食人心智去了
